cmd/url.go

```diff
@@ -0,0 +1,129 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/redhat-developer/ocdev/pkg/application"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/url"
+	"github.com/spf13/cobra"
+)
+
+var (
+	urlListComponent   string
+	urlListApplication string
+)
+
+var urlCmd = &cobra.Command{
+	Use:   "url",
+	Short: "expose component to the outside world",
+	Long:  "expose component to the outside world",
+}
+
+var urlCreateCmd = &cobra.Command{
+	Use:   "create <component name, optional>",
+	Short: "create a URL for a component",
+	Long:  "create a URL for a component",
+	Args:  cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+
+		var cmp string
+		switch len(args) {
```

Comment:
It is good practice to have a default case in every switch.

Reply:
added